.dotnet.azure/src/Custom/VectorStores/AzureCreateVectorStoreOperation.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.ClientModel;
+using System.ClientModel.Primitives;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Azure.AI.OpenAI.VectorStores;
+
+[Experimental("OPENAI001")]
+public class AzureCreateVectorStoreOperation : CreateVectorStoreOperation

```

Comment:
Curious: do these need to be public?

Reply:
If they are internal won't it be hard to test in a separate assembly?

Reply:
This one doesn't have extra functionality yet...